fetch.bs

```diff
@@ -4531,12 +4536,15 @@ Range Requests</cite>. [[HTTP-RANGE]] However, this is not widely supported by b
     <var>httpRequest</var>'s
     <a for=request>credentials mode</a>.
 
-   <li><p>Set <var>httpCache</var> to the result of <a>determining the HTTP cache partition</a>,
+   <li><p>Set <var>httpStatePartition</var> to the result of <a>determining the HTTP state partition</a>,
    given <var>httpRequest</var>.
 
    <li><p>If <var>httpCache</var> is null, then set <var>httpRequest</var>'s
```

Comment:
Since the httpCache is replaced by httpStatePartition above, <var>httpCache</var> here will not be a valid cache?

Reply:
Thanks! Meant to put the line below this above it, but even that doesn't really handle the null case. Fixed.